pyscriptjs/examples/calc.css

```diff
@@ -0,0 +1,109 @@
+body {
+    background-color: #DFDEDE;
```

Comment:
In general, this PR/example, stands out of "native PyScript" look and feel since it adds a very cool UI and, for that, it needs custom CSS _but_ on thing we want to highlight in pyscript is the lower barriers for non-techical users, that don't know css/html/js, etc.. In that sense, would be cool to have a version to don't rely so much on CSS.... (mainly thinking out loud here. Hope it makes sense :) )

Reply:
@fpliger Sure, I will also add one calculator example that uses more of pyscript resources and not rely on CSS. But we have this for people who wants to work with python and HTML,CSS.